pages/cloud-providers.html

```diff
@@ -28,33 +28,18 @@
 
         <div class="card flow-small">
             <div class="card__title">
-                Chère de Prince
+                La Bécasse
             </div>
 
             <p class="text-secondary">
                 French, free
             </p>
 
             <p>
-                <strong><a href="https://cheredeprince.net/">cheredeprince.net</a></strong>
+                <strong><a href="https://rss.cheredeprince.net/">cheredeprince.net</a></strong>
             </p>
         </div>
 
-        <div class="card flow-small">
-            <div class="card__title">
-                FACiL
-            </div>
-
-            <p class="text-secondary">
-                Quebec, pay what you want
-            </p>
-
-            <p>
-                <strong><a href="https://facil.services/">facil.services</a></strong>
```

Comment:
FACiL still provides FreshRSS (see "[Journal FACiLe](https://journal.facil.services/)")

Reply:
Probably I searched in and out, but since I am translated using a translator, maybe I got confused. Will revert that change.